### PR TITLE
new cask for Elephas

### PR DESCRIPTION
**Pull request by alexg0**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/elephas.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+cask "elephas" do
+  # use_beta as a selector if beta or latest version is installed
+  use_beta = true
```
Comment (bevanjkay) [inline Casks/elephas.rb:3]: @alexg0 The version provided in `homebrew-cask` should always be the latest stable version. A beta cask could be added to the `homebrew-cask-versions` repo.

Comment (alexg0) [reply Casks/elephas.rb]: @bevanjkay split into two casks (beta into `homebrew-cask-versions` and removed references to beta from this submission

**Comment by miccal** [comment]
Thank you @alexg0.

**Review by miccal — APPROVED** [review]

**Comment by miccal** [comment]
```
|-> brew audit --cask --appcast --online --new-cask --signing elephas
==> Downloading https://elephas.app/downloads/Elephas_latest.dmg
Already downloaded: /Users/miccal/Library/Caches/Homebrew/downloads/710fa378349be9519152196973eb79d3e1dda4128bfb7633598aa8ea3683a795--Elephas_latest.dmg
Warning: No checksum defined for cask 'elephas', skipping verification.
==> Downloading https://elephas.app/downloads/Elephas_latest.dmg
Already downloaded: /Users/miccal/Library/Caches/Homebrew/downloads/710fa378349be9519152196973eb79d3e1dda4128bfb7633598aa8ea3683a795--Elephas_latest.dmg
Warning: No checksum defined for cask 'elephas', skipping verification.
audit for elephas: passed
```

**Pull request merged**